File: _setToJsongoto.php
<?php
$data = json_decode(file_get_contents('gotoLanguage.json'), true);

echo "Name: " . $data['name'] . "<br>";
echo "Family: " . $data['family'] . "<br>";
echo "Age: " . $data['age'] . "<br>";
echo "Email: " . $data['email'] . "<br>";
echo "Developer: " . $data['developer'];
?>

File: _setToJsongoto.py
import json

data = {
    "name": "Benyamin",
    "family": "Gharri",

    "age": 26,
    "email": "benyamin@gharri.ir",
    "developer": "...."
}

with open('gotoLanguage.json', 'w') as f:
    json.dump(data, f, indent=2)

print("Save data to file system.")
